src/components/taskDetails/TaskDescription.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Textarea } from './index';
+import styles from './task-details.module.scss';
+
+interface TaskDescriptionProps {
+    isEditing: boolean;
+    purpose: string;
+    handleChange: (event: React.ChangeEvent<HTMLTextAreaElement>) => void;
+}
+
+export const TaskDescription: React.FC<TaskDescriptionProps> = ({
+    isEditing,
+    purpose,
+    handleChange,
+}) => {
+    return (
+        <div>
+            {isEditing ? (
+                <Textarea
+                    name="purpose"
+                    value={purpose}
+                    onChange={handleChange}
+                    testId="purpose-textarea"
+                    placeholder="Enter task description"
+                />
+            ) : (
+                <p className={styles.taskDescription}>
+                    {purpose || 'No description available'}
+                </p>
+            )}
+        </div>
+    );
+};
```

Comment:
- move the div to the consumer of this component.
- instead of using ternaries we can use `if` statement to make the code a bit simple to read.
example:
```tsx
if (isEditing){
    return <Textarea ... />
}

return (
    <p>...</p>
)
```